onnxruntime/contrib_ops/cpu/bert/multihead_attention_helper.h

```diff
@@ -42,10 +41,10 @@ Status CheckInputs(const T* query,
   //     value            (V)       : None
   //     bias             (Q/K/V)   : None
   // When packed qkv is used:
-  //     query            (Q)       : (B, L, N, 3, H)
+  //     query            (Q)       : (B, L, N, 3, H) or (B, L, Nx3xH)
```

Comment:
Could we only use 5D for packed QKV?  I think Reshape operator is very cheap to run so you can add a reshape in fusion (see unet fusion as example).

Reply:
I ended up using packed MatMul instead of packed QKV so this case isn't used. I'll remove the logic for packed QKV.